### PR TITLE
Added client to defer creatable

### PR DESCRIPTION
**Pull request by gabrielonilsson**

### CODE REVIEW

File: Payment/Defer/Creatable.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import * as isoly from "isoly"
+import * as authly from "authly"
 import { Item } from "../../Item"
 import { Method } from "./Method"
 
 export interface Creatable {
 	type: "defer"
+	client: authly.Identifier
```
Comment (simonmika) [inline Payment/Defer/Creatable.ts]: Let us make client optional and not only on the `Creatable` version. Only having it there is not very useful. Also we should move it to all payments not only defer payments.

Comment (gabrielonilsson) [reply Payment/Defer/Creatable.ts]: Would that be in Base.ts?

Comment (simonmika) [reply Payment/Defer/Creatable.ts]: Yes but also for all `Creatable` types.

Comment (gabrielonilsson) [reply Payment/Defer/Creatable.ts]: Card, Defer, Installment, Invoice?

Comment (simonmika) [reply Payment/Defer/Creatable.ts]: Yes they currently lack a common base type. Add one.

File: Payment/Defer/index.ts [hunk 2]
```diff
@@ -11,6 +11,19 @@ export namespace Defer {
 		return DeferCreatable.is(value) &&
 			Base.is(value)
 	}
```
Comment (simonmika) [inline Payment/Defer/index.ts]: Move this to all payments. Is this not already merged?

**Pull request merged**